tests/fixtures/todo/basic.hrx

```diff
@@ -0,0 +1,25 @@
+<===> no-todo/input.scss
```

Comment:
HRX style nits: real tests use underscores rather than hyphens in directory names, and end all files with a trailing newline.